mf2py/dom_helpers.py

```diff
@@ -51,6 +51,20 @@ def get_attr(el, attr, check_name=None):
     if isinstance(check_name, (tuple, list)) and el.name in check_name:
         return el.get(attr)
 
+def parse_srcset(srcset, base_url=''):
+    result = {}
+
+    srcset = srcset.split(',')
+
+    for src in srcset:
+        src = src.strip()
+        if not src:
+            continue
+
+        src, size = src.split(' ', 1)
```

Comment:
Looks like this will crash if size (condition) isn't specified. @jgarber623's proposed default condition from https://github.com/microformats/microformats2-parsing/issues/7#issuecomment-1257104515 is `1x`.

Reply:
Yep! `1x` is considered a default descriptor noted in both Mozilla's and the WHATWG's documentation:

> If the condition descriptor is not provided (in other words, the image candidate provides only a URL), the candidate is assigned a default descriptor of "1x".

Source: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset

> For the purpose of this requirement, an image candidate string with no descriptors is equivalent to an image candidate string with a 1x descriptor.

Source: https://html.spec.whatwg.org/multipage/images.html#srcset-attribute